meshroom/ui/qml/GraphEditor/GraphEditor.qml

```diff
@@ -392,6 +392,11 @@ Item {
                         tooltipText: "Iterations"
                         visible: edgeMenu.currentEdge && edgeMenu.forLoop
 
+                        onVisibleChanged: {
+                            if (visible)
+                                value = listAttr.value.indexOf(edgeMenu.currentEdge.src) + 1
```

Comment:
assignations replace/break expressions

Reply:
value: (listAttr && listAttr.value) ? listAttr.value.indexOf(edgeMenu.currentEdge.src) + 1 : 0

Reply:
I tried to change few things